dashboard/src/components/TreeListingPage/TreeListingPage.tsx

```diff
@@ -69,11 +69,15 @@ const TreeListingPage = ({ inputFilter }: ITreeListingPage): JSX.Element => {
             a.branch.localeCompare(b.branch, undefined, { numeric: true }),
         );
     }
-  }, [data, inputFilter]);
+  }, [data, error, inputFilter]);
 
   const { startIndex, endIndex, onClickGoForward, onClickGoBack } =
     usePagination(listItems.length, ITEMS_PER_PAGE);
 
+  if (error) {
+    return <div>Error: {error.message}</div>;
+  }
+
```

Comment:
Please create a ticket to handle this more appropriately

Reply:
https://github.com/kernelci/dashboard/issues/57